ftdetect/slim.vim

```diff
@@ -1 +1 @@
-autocmd BufNewFile,BufRead *.slim setf slim
+autocmd BufNewFile,BufRead *.slim setfiletype slim
```

Comment:
Ah.. I simply prefer longer versions (when it's not making too much noise), as it helps to get overall understanding of what' going on without calling `:help {thing}` even if you're not yet familiar with that call yet...